pkg/provider-local/controller/worker/add.go

```diff
@@ -49,9 +51,17 @@ func AddToManagerWithOptions(mgr manager.Manager, opts AddOptions) error {
 	if err := machinev1alpha1.AddToScheme(scheme); err != nil {
 		return err
 	}
+	clientset, err := kubernetes.NewForConfig(mgr.GetConfig())
+	if err != nil {
+		return err
+	}
+	gardenerClientset, err := kubernetesclient.NewWithConfig(kubernetesclient.WithRESTConfig(mgr.GetConfig()))
+	if err != nil {
+		return err
+	}
```

Comment:
Would be great to perform this initialization inside `NewActuator` instead of doing it in all extensions in a duplicated manner.
Let's rather return an error in `NewActuator` instead of duplicating this boilerplate.